wordssite/dictionary/urls.py

```diff
@@ -1,6 +1,17 @@
-from django.urls import path
+from django.urls import path, reverse
 from . import views
 
+app_name = 'dictionary'
+
 urlpatterns = [
-    path('', views.index, name='index'),
+    path('', views.WordListView.as_view(), name='all'),
+    path('dictionary/<uuid:pk>/detail', views.WordDetailView.as_view(), name='word_detail'),
+    path('dictionary/create/', views.WordCreateView.as_view(), name='word_create'),
+    path('dictionary/<uuid:pk>/update/', views.WordUpdateView.as_view(), name='word_update'),
+    path('dictionary/<uuid:pk>/delete/', views.WordDeleteView.as_view(), name='word_delete'),
+
+    path('', views.TranslationListView.as_view(), name='all'),
```

Comment:
Hm, so empty path leads to two separate views. Should not be like this. We should have one view handling one path, I believe.

Reply:
The `path('', views.TranslationListView.as_view(), name='all'),` is not being used - deleted